internal/metamorph/server.go

```diff
@@ -127,23 +126,25 @@ func (s *Server) Health(ctx context.Context, _ *emptypb.Empty) (*metamorph_api.H
 
 	processorMapSize := s.processor.GetProcessorMapSize()
 
-	peers := s.processor.GetPeers()
+	// TODO: do we need this kind of info? do we check it?
 
-	peersConnected := make([]string, 0, len(peers))
-	peersDisconnected := make([]string, 0, len(peers))
-	for _, peer := range peers {
-		if peer.Connected() {
-			peersConnected = append(peersConnected, peer.String())
-		} else {
-			peersDisconnected = append(peersDisconnected, peer.String())
-		}
-	}
+	// peers := s.processor.GetPeers()
+
+	// peersConnected := make([]string, 0, len(peers))
+	// peersDisconnected := make([]string, 0, len(peers))
+	// for _, peer := range peers {
+	// 	if peer.Connected() {
+	// 		peersConnected = append(peersConnected, peer.String())
+	// 	} else {
+	// 		peersDisconnected = append(peersDisconnected, peer.String())
+	// 	}
+	// }
```

Comment:
It's not my call, but IMO we can remove it. But maybe, it would be useful to mark each peer in p2p as `healthy`/`unhealthy` anyways in go-p2p? I think that might be useful. WDYT?

@arkadiuszos4chain @boecklim

Reply:
In the new implementation, there is no need to distinguish between a connected peer and a healthy one. If the peer is connected, it is healthy. If it becomes unhealthy (error, ping-pong failed), it will disconnect and wait for the client(owner) to initiate a reconnect